esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/flow.cylc

```diff
@@ -3,17 +3,20 @@
     UTC mode = True
 
 [task parameters]
-    fast = radiation_budget, albedolandcover
-    medium = ensclus, heatwaves_coldwaves, autoassess_landsurface_soilmoisture
-
+    # `fast` recipes are those taking less than 10 mins.
+    # Other recipes should be included in `medium`
+    fast = radiation_budget, albedolandcover, heatwaves_coldwaves, \
+           autoassess_landsurface_soilmoisture
+    medium = ensclus  # Should be in fast: is in medium temporarily to
+                      # avoid empty variable
 
 [scheduling]
     initial cycle point = {{ START_DATE }}T{{ RUN_TIME }}
```

Comment:
Just found out this can be set to `now` to start it with the current date and time (which saves failed tasks if running with an earlier date)! 💪 Would that be better? 🤔

Reply:
This is very cool! Implemented in [33a88ba](https://github.com/ESMValGroup/ESMValTool/pull/3562/commits/33a88bae8737038743ec6394e8bd3526d3805952) (with tidying in [b3e47a7](https://github.com/ESMValGroup/ESMValTool/pull/3562/commits/b3e47a77cd801e675ed0dd956ee283f1d134289d)